tasks/internal_reconcile_report_phantom/schemas/input.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/internal_reconcile_report_phantom/schemas/input.json",
+    "title": "internal_reconcile_report_phantom Lambda input",
+    "description": "The input for the internal_reconcile_report_phantom Lambda.", 
+    "type": "object",
+    "required": [
+      "reportId", "pageNumber"
+    ],
+    "properties": {
+      "reportId": {
+        "type": "string",
+        "description": "The report ID requested by user for the report."
+      },
+      "pageNumber": {
+        "type": "integer",
+        "description": "The page number of the report to return that has been requested by user."
+      }
```

Comment:
Does there also need to be an optional page_size or page_limit? Maybe talk with @AndrewDorn on how he wants to implement paging.